PENDING_RELEASE_NOTES.md

```diff
@@ -4,6 +4,7 @@ _Date TBD_
 
 ### New Features
 * Update ONC contact information for CHPL API and on user invitation emails.
+* 2015 Functionality testing is restricted by criteria.
```

Comment:
This says 2015, but there's code below that talks about 2014. Does it also restrict 2014 Listings?

Reply:
2014 listings were previously restricted by criteria.  It just so happened that for 2014 there was only one criteria for each test functionality.  The change for 2014 test functionalities is transparent to the users.  The data that had to be migrated for the 2014 to work with the new table structure has been noted in the release notes for the data model project.